tests/integration/tests/autocomplete.js

```diff
@@ -45,6 +45,13 @@ test('search and clear', async () => {
   expect(searchValueAfterClear).toEqual('');
 });
 
+test('search with no suggest', async () => {
+  responseHandler.addPreparedResponse(mockAutocompleteEmpty, /autocomplete\?q=Goodbye/);
+  await page.goto(APP_URL);
+  await autocompleteHelper.typeAndWait('Goodbye');
+  await page.waitForSelector('.autocomplete_suggestion--no-result');
```

Comment:
Tests should end with an `expect`, otherwise when they fail they exit with a dirty, hard-to-read error (`Target closed`, etc.), instead of clear message explaining the difference between what we want and what we got.
For existence tests, you can use
```js
expect(await exists(page, '.autocomplete_suggestion--no-result')).toBeTruthy();`
```